packages/opentelemetry-cloud-monitoring-exporter/src/monitoring.ts

```diff
@@ -140,7 +140,7 @@ export class MetricExporter implements IMetricExporter {
     cb(ExportResult.SUCCESS);
   }
 
-  shutdown(): void {}
+  async shutdown(): Promise<void> {}
```

Comment:
just curious, where did this change come from?

Reply:
They changed the interface in the SDK: https://github.com/open-telemetry/opentelemetry-js/pull/1439